test/ng/compileSpec.js

```diff
@@ -3137,6 +3137,54 @@ describe('$compile', function() {
     });
 
 
+    it('should support custom start interpolation symbol, even when `endSymbol` doesn\'t change',
```

Comment:
I know, I just thought it was more explicit to state that the other symbol remains the default symbol. As is works for me though.